web-app/src/main/webapp/index.html

```diff
@@ -132,85 +132,48 @@ <h2>Category</h2>
       <div class="row">
         <div class="two columns right">
           <input type="checkbox" class="category" id="p-one" name="category"
-            value="drugs" checked="checked">
+            value="Theft Crimes" checked="checked">
```

Comment:
Drop the word crime here? It's a bit redundant

Reply:
Can't drop crimes for value because I will have to do preprocessing again, but dropped it for what is going to be displayed for users!